aten/src/ATen/cuda/detail/OffsetCalculator.cuh

```diff
@@ -7,8 +7,9 @@
 #include <ATen/native/TensorIterator.h>
 #include <THC/THCIntegerDivider.cuh>
 
-/// OffsetCalculator calculates the offset in bytes of a linear index for NARGS
-/// operands that share the same shape, but may have different strides.
+// OffsetCalculator calculates the offset in bytes of a linear index for NARGS
```

Comment:
Not introduced by your PR, but in the common case of element_sizes being nullptr, OffsetCalculator calculates offset in elements, can you please leave a comment to that effect here?